api/src/main/java/ca/bc/gov/educ/api/gradstudent/controller/CommonController.java

```diff
@@ -275,9 +286,9 @@ public ResponseEntity<List<String>> getSchoolReportDataForYearEndNonGrad() {
             description = "Find Schools for Year End School Report", tags = {"Schools for Year End School Reports"})
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "OK"),
             @ApiResponse(responseCode = "204", description = "NO CONTENT.")})
-    public ResponseEntity<List<String>> getDistrictReportDataForYearEndNonGrad() {
+    public ResponseEntity<List<UUID>> getDistrictReportDataForYearEndNonGrad(@RequestHeader(name="Authorization") String accessToken) {
         logger.debug("getSchoolReportDataForYearEndNonGrad :");
-        return response.GET(gradStudentReportService.getGradDistrictsForNonGradYearEndReport());
+        return response.GET(gradStudentReportService.getGradDistrictsForNonGradYearEndReport(accessToken.replace(BEARER, "")));
```

Comment:
Why use access token? We should try to get rid of these

Reply:
This can be handled in a new ticket.